src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -237,6 +237,18 @@ public String getAwsSecretKey() {
     public String getAwsEndpoint() {
         return getString("aws.endpoint");
     }
+    
+    public boolean getAwsProxyEnabled(){
+    	return getBoolean("aws.proxy.isEnabled", false);
```

Comment:
We usually want the config default specified in properties file, there is no need to provide a default in java code.